### DISCUSSION TITLE
How can i set up a repository to collaborate with a small team

### DISCUSSION THREAD

**Discussion by slippin-jimm** [community/community#18170] [Pull Requests] [answered]
I would like to be the one in charge of merging the features developed on other branches to the main branches.

**Comment by david-s-martinez** [comment, marked as answer]
hey , I found a guide on [medium ](https://medium.com/@jonathanmines/the-ultimate-github-collaboration-guide-df816e98fb67#:~:text=Click%20on%20the%20%E2%80%9CSettings%E2%80%9D%20tab,be%20listed%20as%20a%20collaborator.)that explains what you need. Basically you have to agree with your teammates who will merge all the changes on other branches and then you will select that person as the reviewer when submitting the pull request.

**Reply by slippin-jimm** [reply]
Awesome! Thought it would be harder. Thanks for the fast reply!
